controller/src/main/java/com/softserve/borda/controllers/AuthController.java

```diff
@@ -29,22 +31,28 @@
     private final ModelMapper modelMapper;
 
     @PostMapping("/register")
-    public ResponseEntity<AuthResponse> registerUser(@RequestBody RegistrationRequest registrationRequest)
+    public ResponseEntity<AuthResponse> registerUser(@RequestBody @Valid RegistrationRequest registrationRequest,
+                                                     BindingResult bindingResult)
             throws CustomAuthenticationFailedException {
         try {
-            User user = new User();
-            user.setUsername(registrationRequest.getUsername());
-            user.setPassword(passwordEncoder.encode(registrationRequest.getPassword()));
-            user.setEmail(registrationRequest.getEmail());
-            user.setFirstName(registrationRequest.getFirstName());
-            user.setLastName(registrationRequest.getLastName());
-            userService.create(user);
+            if (bindingResult.hasErrors()) {
```

Comment:
I think is it unnecessarily

Reply:
With bindingResult, I store all errors that occur during validation and then pass them to my customValidationFailedException. 
Then this exception log it in format like :  Validation failed in field: [firstName] because value: [@!@!##!@] doesn't match the condition: [First name can only contain letters and should start with Upper case].
I did this I did this to display the errors more clearly. Should i remove that?